src/Core/Resolvers/MySqlQueryBuilder.cs

```diff
@@ -323,6 +353,14 @@ private static string GetMySQLDefaultValue(ColumnDefinition column)
             return defaultValue;
         }
 
+        /// <inheritdoc/>
+        public string BuildQueryToGetReadOnlyColumns(string schemaParamName, string tableParamName)
+        {
+            string query = "select column_name as column_name from information_schema.columns " +
```

Comment:
```suggestion
            string query = "select column_name from information_schema.columns " +
```

Is `column_name as column_name` necessary since you aren't changing the name of the returned column?

Reply:
Yes, its needed. By default, with no alias, the name is captialized as `COLUMN_NAME`. So, I aliased it to `column_name`.